packages/orchestration/src/cosmos-api.ts

```diff
@@ -317,17 +328,15 @@ export interface IBCMsgTransferOptions {
  *
  * @see {OrchestrationAccountI}
  */
-export type CosmosChainAccountMethods<CCI extends CosmosChainInfo> =
+export type CosmosChainAccountMethods<CCI extends CosmosChainInfo> = Pick<
```

Comment:
this `Pick` looks arbitrary. I thought maybe it was `IcaAccount` without the getters, but it also omits `executeTx`. Is that because it's not yet implemented? Not a good reason when defining types.

Whatever the criteria are for this set of methods, please name it as a type and document it there. Alternately define it as an interface that `IcaAccount` extends.

Reply:
An interface that `IcaAccount` extends makes sense to me. Do you have a suggestion for the name of the type? `IcaAccountBase(Methods)` is the best I've come up with so far.

Regarding `executeTx` - I suggest we remove this from `IcaAccount` until there's a story that requires it. The original idea was to keep it to support Proto3Json messages, but it doesn't seem any chain has adopted this yet for ICAs. Having `executeTx` show up in typeahead in a flow for a non-agoric account seems like something we should avoid. cc @dtribble

RE the other methods omitted - 
- `getAddress` already appears on `OrchestrationAccountI`
- `getRemoteAddress`, `getLocalAddress`, and `getPort` seem like lower level things an orchestration developer shouldn't be concerned with. However, if we feel these should be lifted to the `CosmosOrchAccount` exo, I wouldn't be opposed

Reply:
"Base" is a smell to me. How about `IcaActions`? I agree the getters shouldn't be available in an Orchestration flow.

+1 to removing `executeTx` until we have a story that requires it.

Reply:
> +1 to removing executeTx until we have a story that requires it.

It turns out this is a [breaking change](https://github.com/Agoric/agoric-sdk/commit/22dfc5f21fa82b88378ac3fddb5d0a3be224b8e0) for `IcaAccountKit` which was deployed as part of `vat-orchestration` in u17. The upgrade considerations seem low since consumers do not depend on it, but I'll layer it in a separate PR.